### PR TITLE
Story/vgnwb 93 - Enabled sort by asc and desc on Text table columns

### PR DESCRIPTION
**Pull request by erhkamal**
Hi Erick,

Could you please review and make a comment?

Thanks,
Kamal & Harshaneel.

### CODE REVIEW

File: annotations/templates/annotations/list_texts.html [hunk 1]
```diff
@@ -22,13 +22,13 @@
         <div class="col-sm-8 col-sm-offset-2">
             {% if texts|length > 0 %}
             <table class="table table-hover">
-              <th class="col-sm-8">Text title</th>
-              <th class="col-sm-3">Publication date</th>
+              <th class="col-sm-8"><a href="{% if order_by == '-title' %}?order_by=title{% else %}?order_by=-title{% endif %}" id="order_title">Text title</a></th>
```
Comment (erickpeirson) [inline annotations/templates/annotations/list_texts.html]: What happens when I'm on page 4, and I click on this link?

Comment (erhkamal) [inline annotations/templates/annotations/list_texts.html:25]: goes back to page 1 and toggles the sort order.

**Comment by erickpeirson** [comment]
Actually, ignore that last comment. It makes sense that we would not preserve the page number here.

**Comment by erhkamal** [comment]
Yeah, thanks.

**Pull request merged**